docs/conditions.md

```diff
@@ -6,99 +6,7 @@ weight: 2100
-    this `Condition` resource object. This must include:
-    - [`check`](#check) - Specifies a container that you want to run for evaluating this `Condition`.
-    - [`description`](#description) - Provides a meaningful description of this `Condition` object.
-
-[kubernetes-overview]:
-  https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
-
-### Specifying the condition `check` 
-
-The `check` field (required) specifies a single piece of evaluation logic that you want to run before the
-corresponding `Task` in your `Pipeline` can execute. This field must specify a [`Step`](./tasks.md#steps). 
-
-### Specifying `Parameters`
-
-You can specify parameters to pass to the `Condition's` evaluation logic at run time. 
-Sub-fields within the `check` field can access these parameter values using Tekton's templating
-syntax as follows:
-
-```yaml
-spec:
-  parameters:
-    - name: image
-      default: ubuntu
-  check:
-    image: $(params.image)
-```
-
-Parameter names:
-- Must only contain alphanumeric characters, hyphens (`-`), and underscores (`_`).
-- Must begin with a letter or an underscore (`_`).
-
-For example, `fooIs-Bar_` is a valid parameter name, but `barIsBa$` or `0banana` are not.
-
-Each declared parameter has a `type` field, which can be set to either `array` or `string`, and
-defaults to `string` if you don't specify a value. The `description` and `default` fields for a
-`Parameter` are optional. The `array` type is useful in situations such as checking that a pushed
-branch name doesn't collide with any of the specified protected branch names.
-
-### Specifying `Resources`
-
-> :warning: **`PipelineResources` are [deprecated](deprecations.md#deprecation-table).**
->
-> Consider using replacement features instead. Read more in [documentation](migrating-v1alpha1-to-v1beta1.md#replacing-pipelineresources-with-tasks)
-> and [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md).
-
-You can specify input [`PipelineResources`](resources.md) in your `Condition` definition to 
-provide the `Condition's` container step with data or context necessary to run the evaluation logic.
-
-`Resources` in `Conditions` behave the same way as in `Tasks`:
-- You can access them via [variable substitution](resources.md#variable-substitution).
-- You can use the `targetPath` field to [specify a mount point](resources.md#controlling-where-resources-are-mounted).
-
-### Adding a `description`
-
-The `description` field (optional) allows you to specify a meaningful description for your `Condition`.
-
-## Code examples
-
-For a better understanding of `Conditions`, study [our code examples](https://github.com/tektoncd/pipeline/tree/main/examples).
+**Note:** `Conditions` have been removed as of Pipeline 0.37.0, use [`when` expressions](pipelines.md#guard-task-execution-using-when-expressions) instead. 
```

Comment:
I think we can just remove this entirely

Reply:
I thought it'd be worth keeping this for search results purposes.